### DISCUSSION TITLE
Still unable to re-verify my academic affiliation as a faculty

### DISCUSSION THREAD

**Discussion by gaomy3832** [community/community#54126] [GitHub Education]
### Hi everyone,

I am a faculty at Tsinghua University in Beijing, China. I submitted an issue ticket about a month ago (https://github.com/orgs/community/discussions/51810) about being unable to re-verify my academic affiliation as the coupon is expiring. I am still facing the same problem now, after a month of trials and 30+ requests. I have fixed all the issues including VPN, English translation, formal letter from mobile phone camera, etc. Again, it simply says the following general information without telling me why.

I am also disappointed that my last ticket has NO RESPONSE from the official maintainers. Please at least respond and let me know whom to reach.

@samus-aran I see you have been kindly responding to most of the similar issues in this discussion forum. Could you please comment? Thanks!

![Screen Shot 2023-05-01 at 11 49 15 AM](https://user-images.githubusercontent.com/3386287/235403873-9a09c204-39d3-4ad5-9849-c7cfa58c04fd.png)

**Comment by samus-aran** [comment]
Hi @gaomy3832 👋 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by gaomy3832** [reply]
Many thanks for the help! I will be looking forward to any update.

In the meantime, since my previous coupon expires on May 2nd (today), is there anything that can be done to fill in the gap? To avoid service degrade, I have already put in a credit card in case a payment needs to be billed to me.

**Reply by gaomy3832** [reply]
@samus-aran Hi, I just would like to check back if there is any progress. I tried to reverify today (June 15) but still no luck.

This is important because my own personal account and some organization accounts associated with me start to bill. These amounts of cost should have been avoided if I can smoothly reverify my status. Is there work-around such as some coupons to be issued to me before the issue can be resolved?

**Comment by samus-aran** [comment]
Hi @gaomy3832 ,

This post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

We can continue [your conversations in this discussion](https://github.com/orgs/community/discussions/58126). Thank you for helping us maintain a productive and tidy community for all our members.
